conda/environments/all_cuda-118_arch-x86_64.yaml

```
@@ -40,7 +40,7 @@ dependencies:
 - hypothesis
 - identify>=2.5.20
 - ipython
-- libarrow==12.0.1.*
+- libarrow==12.0.1.*=hb6645e0_10_cpu
```

Think one `=`s (instead of 2 `=`s) has the behavior of allowing `.*` when relevant. Idk if the `==`s is also causing us issues finding the intended version (hence mentioning this here)

```suggestion
- libarrow=12.0.1
```